dpnp/backend/extensions/lapack/syevd.cpp

```diff
@@ -230,6 +230,136 @@ std::pair<sycl::event, sycl::event>
     return std::make_pair(args_ev, syevd_ev);
 }
 
+std::pair<sycl::event, sycl::event>
+    syevd_batch(sycl::queue &exec_q,
```

Comment:
`heevd_batch` and `syevd_batch` looks exactly the same, isn't they?
The only sensible difference is array of callbacks, but it can be passed as an input argument.

Reply:
Yes, another difference is that we use **...dispatch_table** and **...dispatch_vector** to get a pointer to the executing function